.azure-pipelines/tests.yml

```diff
@@ -15,9 +15,9 @@ variables:
 jobs:
   - job: 'Test'
 
-    # Increase timeout so Event Hubs tests do not timeout
+    # Increase timeout so Event Hubs tests do not timeout (Windows runs for ~2h 25m, on average due to having two target platforms)
     # https://github.com/Azure/azure-sdk-for-net/issues/5982
-    timeoutInMinutes: 90
+    timeoutInMinutes: 155
```

Comment:
:(

@chidozieononiwu we should up the priority of https://github.com/Azure/azure-sdk-for-net/issues/5776 while we wanted to separate for reporting it is also a good idea to separate for runtime performance.

Reply:
Yeah.  This makes me really sad as well.  My goal is to get these isolated, parallel, and running under 10 minutes, but that may be further out than we'd like due to other priorities.